### DISCUSSION TITLE
Pre-release version of package

### DISCUSSION THREAD

**Discussion by pvgritsenko-ansible** [Jericho/ZoomNet#377] [General]
Hi @Jericho. I'm not sure that is best place to ask, but I've not found any other ways to contact you. Could you please prepare pre-release version of ZoomNet package? I want to test the GetPresenceStatusAsync and UpdateCallHandlingSettingsAsync endpoints before we will push them into new release. Especially the UpdateCallHandlingSettingsAsync because it has very complicated payload model.

**Comment by Jericho** [comment]
Good idea. Give me an hour or so and it will be on my personal NuGet feed ([instructions](https://github.com/Jericho/Jericho/blob/main/MyNugetFeed.md)).

Looking forward to your feedback.

**Reply by pvgritsenko-ansible** [reply]
Great! Thanks a lot!
I'll let you know about the test results ASAP.
